src/jit/SimdInl.h

```diff
@@ -177,8 +201,16 @@ static void emitSplatSIMD(sljit_compiler* compiler, Instruction* instr)
     JITArg args[2] = { operands + 0, operands + 1 };
     sljit_s32 dstReg = GET_TARGET_REG(args[1].arg, SLJIT_FR0);
 
-    sljit_emit_simd_replicate(compiler, type, dstReg, args[0].arg, args[0].argw);
-
+#if (defined SLJIT_CONFIG_ARM_32 && SLJIT_CONFIG_ARM_32)
+    if (type == (SLJIT_SIMD_REG_128 | SLJIT_SIMD_ELEM_64)) {
+        sljit_emit_simd_mov(compiler, SLJIT_SIMD_LOAD | SLJIT_SIMD_REG_64 | SLJIT_SIMD_ELEM_8, SLJIT_FR0, args[0].arg, args[0].argw);
+        sljit_emit_simd_mov(compiler, SLJIT_SIMD_LOAD | SLJIT_SIMD_REG_64 | SLJIT_SIMD_ELEM_8, SLJIT_FR1, args[0].arg, args[0].argw);
```

Comment:
Do not load twice, just copy the loaded value. And use dstReg / getHighRegister(dstReg). I think I said this before.